docs/logger.rst

```diff
@@ -169,3 +177,20 @@ You can use predefined celery task ``security.tasks.call_django_command`` to run
 
     Is the message which will be logged as warning if task is failed. Default value is ``'Task "{task_name}" ({task}) failed on exception: "{exception}"'``
 
+  .. property:: unique
+
+    LeggedTask can guarantee unique celery task. It means that only one task with same name and input can run at one time. If task is already running is its AsyncResult is returned in the methods ``apply_async``, ``apply_async_on_commit``, ``delay`` or     LeggedTask can guarantee unique celery task. It means that only one task with same name and input can run at one time. If task is already running is its ``AsyncResult`` is returned in methods ``apply_async``, ``apply_async_on_commit``, ``delay`` or ``delay_on_commit``.
```

Comment:
LoggedTask can guarantee unique celery task. It means that only one task with the same name and input can run at one time. If task is already running its AsyncResult is returned in the methods ``apply_async``, ``apply_async_on_commit``, ``delay`` or  LoggedTask can guarantee unique celery task.